load_tests/task_definitions/cloudwatch.json

```diff
@@ -0,0 +1,50 @@
+{
+	"family": "load-test-fluent-bit-cloudwatch-$THROUGHPUT",
+	"taskRoleArn": "$TASK_ROLE_ARN",
+	"executionRoleArn": "$EXECUTION_ROLE_ARN",
+	"containerDefinitions": [
+		{
+			"essential": true,
+			"image": "906394416424.dkr.ecr.us-east-1.amazonaws.com/aws-for-fluent-bit:stable",
```

Comment:
Needs to be replaced by our testing image (not available for ECS tasks). Want to discuss offline about the solution here.